File: AndroidApp/PayIn/app/src/main/java/com/mobile/application/payin/dto/arguments/DeviceMobileCreateArguments.java
package com.mobile.application.payin.dto.arguments;

public class DeviceMobileCreateArguments {
    public String Token;
    public int Type;

    public DeviceMobileCreateArguments() {
    }

    public static class DeviceType {
        public static final int ANDROID = 1;
    }
}

File: BusinessLogic/PayIn.Application.Dto/Arguments/Device/DeviceMobileCreateArguments.cs
using PayIn.Common;
using System;
using System.Collections.Generic;
using System.Text;
using Xp.Common.Dto.Arguments;

namespace PayIn.Application.Dto.Arguments.Device
{
    public class DeviceMobileCreateArguments : IArgumentsBase
    {
        public string Token {get; set;}
        public DeviceType Type { get; set;}

		public DeviceMobileCreateArguments(string token, DeviceType type)
		{
			Token = token;
			Type = type;
		}
    }
}
